README.md

```diff
@@ -22,6 +22,9 @@ This repository is started to celebrate [Hacktoberfest 2020](https://hacktoberfe
   - [Database Management Systems](#database-management-systems)
   - [Artificial Intelligence](#artificial-intelligence)
   - [Computer Graphics](#computer-graphics)
+
+- [Open Source Projects](#open-source-projects)
+  - [VS Code Extensions](#extensions)
```

Comment:
I don't think we will be having more vs code extensions, can you change the VS code Extensions title to your project name? Keeping it under Open source projects